libs/core/src/lib/grid-list/components/grid-list-item/grid-list-item.component.html

```diff
@@ -199,6 +219,8 @@
                 label="Draft"
                 [compact]="true"
                 (click)="_clickOnDraft($event)"
+                (mouseover)="_isActive = false"
+                (mouseout)="_isActive = true"
```

Comment:
I don't know which rules applied in current project, but on my previous project we had a rule: don't change state of component inside template.

Reply:
already updated